data-visualization/pages/03_Community_evolution.py

```diff
@@ -8,10 +8,16 @@
 
     Comparison:
         A public comparison between two videos involves one or more quality criteria.
+
+    Active contributor (during a period of time):
+        An active contributor is a user who makes at least one public comparison during a given
+        period of time.
 """
 
 import pandas as pd
+import plotly.express as px
 import streamlit as st
+from dateutil.relativedelta import relativedelta
```

Comment:
This is not a direct dependency of the project, should we add it to the file `requirements.txt` ?